tests/drum/test_custom_model.py

```diff
@@ -480,30 +482,32 @@ def wait_for_server(url, timeout, process_holder):
                 assert timeout, "Server failed to start: url: {}".format(url)
 
     @pytest.mark.parametrize(
-        "framework, problem, language, docker",
+        "framework, problem, language, docker, nested",
         [
-            (SKLEARN, REGRESSION, PYTHON, DOCKER_PYTHON_SKLEARN),
-            (SKLEARN, BINARY, PYTHON, None),
-            (KERAS, REGRESSION, PYTHON, None),
-            (KERAS, BINARY, PYTHON, None),
-            (XGB, REGRESSION, PYTHON, None),
-            (XGB, BINARY, PYTHON, None),
-            (PYTORCH, REGRESSION, PYTHON, None),
-            (PYTORCH, BINARY, PYTHON, None),
-            (RDS, REGRESSION, R, None),
-            (RDS, BINARY, R, None),
-            (CODEGEN, REGRESSION, NO_CUSTOM, None),
-            (CODEGEN, BINARY, NO_CUSTOM, None),
-            (MULTI_ARTIFACT, REGRESSION, PYTHON_LOAD_MODEL, None),
-            (PYPMML, REGRESSION, NO_CUSTOM, None),
-            (PYPMML, BINARY, NO_CUSTOM, None),
+            (SKLEARN, REGRESSION, PYTHON, DOCKER_PYTHON_SKLEARN, True),
+            (SKLEARN, BINARY, PYTHON, None, False),
+            (KERAS, REGRESSION, PYTHON, None, False),
+            (KERAS, BINARY, PYTHON, None, True),
+            (XGB, REGRESSION, PYTHON, None, False),
+            (XGB, BINARY, PYTHON, None, False),
+            (PYTORCH, REGRESSION, PYTHON, None, False),
+            (PYTORCH, BINARY, PYTHON, None, False),
+            (RDS, REGRESSION, R, None, True),
+            (RDS, BINARY, R, None, False),
+            (CODEGEN, REGRESSION, NO_CUSTOM, None, False),
+            (CODEGEN, BINARY, NO_CUSTOM, None, False),
+            (MULTI_ARTIFACT, REGRESSION, PYTHON_LOAD_MODEL, None, False),
+            (PYPMML, REGRESSION, NO_CUSTOM, None, True),
+            (PYPMML, BINARY, NO_CUSTOM, None, False),
```

Comment:
what's the logic for what is False and what is True?

Reply:
it should work for the true ones. I'm kinda just letting it spin, but I don't think nested dirs is gon work for java and R for now...

Reply:
> what's the logic for what is False and what is True?

+1 : create a separate usecase, THen you won't need to put these dummy False.

And then, ode day, we could add R.